Podfile

```diff
@@ -14,6 +14,7 @@ def all_test_pods
   pod 'SDWebImage/MapKit', :path => './'
   pod 'Expecta'
   pod 'KVOController'
+  pod 'SDWebImageWebPCoder', :git => 'https://github.com/SDWebImage/SDWebImageWebPCoder.git', :branch => 'master'
```

Comment:
...Why add WebP coder back to Core repo ?😂

WebP coder related tests should be placed into `SDWebImageWebPCoder` repo, not here.

If you need to test WebP decoding of SD, since SD use `SDWebImageAWebPCoder`, which is built of ImageIO, you can just use that instead. The Travis CI use iOS 14 simulator which supports WebP with Imageio.

Reply:
I'll have a try in `SDWebImageAWebPCoder `